Si702x.class.nut

```diff
@@ -13,61 +16,96 @@ class Si702x {
     static TIMEOUT      = 200; // ms
```

Comment:
In general, I prefer that the units be in the variable/constant name, so `TIMEOUT_MS`.